Detector/DetFCChhTrackerSimple/src/GenericBarrel_Geo.cpp

```diff
@@ -0,0 +1,161 @@
+
+#include "DetCommon/DetUtils.h"
+
+#include "DD4hep/DetFactoryHelper.h"
+
+using DD4hep::Geometry::Volume;
+using DD4hep::Geometry::DetElement;
+using DD4hep::XML::Dimension;
+using DD4hep::Geometry::PlacedVolume;
```

Comment:
But I am using the `using` statements, e.g. https://github.com/HEP-FCC/FCCSW/pull/115/files#diff-6739f93c51529d9fd488b8cd20bc5214R30